src/client/rest_client/projectsClient.js

```diff
@@ -154,7 +154,7 @@ class ProjectsClient extends BaseClient {
     getLatestCommits(ownerId, projectName, numCommits) {
         var query;
         numCommits = numCommits || 100;
-        query = '?n=' + numCommits;
+        query = 'commits?n=' + numCommits;
         return super.get('projects/' + ownerId + '/' + projectName, query);
```

Comment:
I think it has to be in the query because the base client adds a '/' between the 
path and the query at the moment